crypto/src/note.rs

```diff
@@ -333,6 +333,24 @@ pub struct Commitment(pub Fq);
 
 impl Protobuf<pb::NoteCommitment> for Commitment {}
 
+impl From<Commitment> for Fq {
+    fn from(commitment: Commitment) -> Fq {
+        commitment.0
+    }
+}
+
+impl From<Commitment> for penumbra_tct::Commitment {
+    fn from(commitment: Commitment) -> penumbra_tct::Commitment {
+        commitment.0.into()
+    }
+}
+
+impl From<penumbra_tct::Commitment> for Commitment {
+    fn from(commitment: penumbra_tct::Commitment) -> Commitment {
+        Commitment(commitment.into())
+    }
+}
```

Comment:
Having two copies of the (logically) same type in two different crates is pretty awkward, although I guess it's not the worst thing for now.